enterprise/models.py

```diff
@@ -261,6 +262,13 @@ class Meta:
         default='',
         help_text=_("Hostname of the enterprise learner portal, e.g. bestrun.edx.org.")
     )
+    catalog_uuids = models.TextField(
```

Comment:
This field should in `EnterpriseCustomerPluginConfiguration` model and its name should be `catalogs_to_transmit`